src/GovUk.Education.ExploreEducationStatistics.Content.Model/Methodology.cs

```diff
@@ -73,12 +73,17 @@ public bool ScheduledForPublishingWithPublishedRelease
         {
             get
             {
+                if (PublishingStrategy != MethodologyPublishingStrategy.WithRelease)
+                {
+                    return false;
+                }
+                
                 if (ScheduledWithReleaseId != null && ScheduledWithRelease == null)
                 {
                     throw new InvalidOperationException("ScheduledWithRelease field not included in Methodology");
                 }
-                return PublishingStrategy == MethodologyPublishingStrategy.WithRelease
-                       && ScheduledWithRelease.Live;
+                
+                return ScheduledWithRelease.Live;
```

Comment:
Is there a potential null reference exception here if `PublishingStrategy` is `WithRelease` but the ScheduledWithReleaseId isn't set? Looks like it would fall through to evaluating `ScheduledWithRelease.Live` with a null `ScheduledWithRelease`.

I don't think this matters though as `ScheduledWithReleaseId` should always be set with that strategy.

Reply:
Agreed :)